docs/source/noise/index.rst

```diff
@@ -38,45 +39,72 @@ Currently, errors added in different columns are independent of each other.
 Available Noise Types
 ---------------------
 
-Pseudopeople applies the different types of row-based and column-based noise in
-the following order to mimic the data generation process by which a real dataset
-might be corrupted. The table lists all the available noise types, but not every
+The table lists all the available noise types, but not every
 type of noise will necessarily be applied to every dataset or every column. The
 configuration determines which noise types are actually used. The "Noise Type"
 column shows the name of the noise type in the configuration system.
 
-.. list-table:: Types of noise in order of application
-  :widths: 1 2 5 1
+.. list-table:: Types of row-based noise (``row_noise``)
+  :widths: 1 2 5 
   :header-rows: 1
 
   * - Noise Type
-    - Description
-    - Example
-    - Order
-  * - ``row_noise``
-    - **Row-based noise**
-    - ---
-    - ---
-  * - ``omit_row``
-    - Omit a row
+    - Config key 
+    - Example cause
+  * - Omit a row
+    - ``omit_row``
     - Neglecting to file a tax form on time
-    - 1
-  * - ``column_noise``
-    - **Column-based noise**
-    - ---
-    - ---
-  * - ``leave_blank``
-    - Leave a field blank
+  * - Duplicate a row
+    - ``duplicate_row``
+    - College student being reported both at their university and their guardian's
+      home address
+
+
+.. list-table:: Types of column-based noise (``column_noise``)
+  :widths: 1 2 5 
+  :header-rows: 1
+
+  * - Noise Type
+    - Config key
+    - Example cause
+  * - Leave a field blank
+    - ``leave_blank``
     - Forgetting to write your name on the designated line
-    - 2
-  * - ``choose_wrong_option``
-    - Choose the wrong option from a fixed set of options
+  * - Choose the wrong option from a fixed set of options
+    - ``choose_wrong_option``
     - Marking the "Male" box when you meant "Female"
-    - 3
-
-.. todo::
+  * - Optical character recognition (OCR) error
+    - ``make_ocr_errors``
+    - Misreading an 'S' instead of a '5'
+  * - Phonetic error
+    - ``make_phonetic_errors``
+    - Mishearing a 't' for a 'd'
+  * - Typographic errors
+    - ``make_typos``
+    - Accidentally typing an 'l' instead of a 'k' because they are 
+      right next to each other on a QWERTY keyboard
+  * - Nicknames
+    - ``use_nickname``
+    - Writing 'Alex' instead of legal name 'Alexander'
+  * - Fake names
+    - ``use_fake_name``
+    - Using fake name rather than official name
+  * - Numeric miswriting 
+    - ``write_wrong_digits``
+    - Listing incorrect date of birth 
+  * - Age miswriting
```

Comment:
I don't think this name matches with what is on the noise types page, can you check these?